### ISSUE TITLE
Feature request: make time and date more distinct on the "recent calls" screen

### ISSUE THREAD

**Issue by genodeftest** [SimpleMobileTools/Simple-Dialer#33]
Steps to reproduce:
1. Set the date format to `dd.mm.yyyy`
2. Have a look at the call history

What happens:
the call history shows date and time of old calls, e.g.  `20:05` and `20.05`. In this case, it is hard to distinguish date and time by a first look.

What should happen:
Have different date/time format which makes it easier to distinguish date and time. Example:
* today, 8:05
* yesterday, 20:05
* Wednesday
* Monday
* 12.5.

In other words: For the last 24 hours (or today and yesterday), specify the date by the word "today" or "yesterday" respectively. For the other days within the last 7 days, use weekdate name plus date. For older days, use the date only.

Or maybe swap date word and time:
* 8:05 today
* 20:05 yesterday
* Wednesday 20.05.
* Monday 18.05.
* 12.05.

Version info:
5.2.0

**Comment by qw123wh** [comment]
Hello is more or less what I asked too, Tibbi unifies the two problems.

https://github.com/SimpleMobileTools/Simple-Dialer/issues/10

**Comment by pvagner** [comment]
I like relative times where they make sense too.
So if the entry in the call history is less than a hour old it should say %d minutes ago. If it's less than 24 hours old it should say %d hours ago.
Then yesterday at HH:MM.
Then Day name at HH:MM for entries that are up to a week old.
and then configured date format for older entries.
Perhaps there is already a library for doing this taking proper localization into account. Many social network client apps are doing this these days.

**Comment by meskobalazs** [comment]
I would also add, that the strings make proper Hungarian translation difficult. I have two issues:

- The name of the days and month should only be capitalized at the start of the label, never in the middle (so these should be separate).
- The date formats are also not a good fit for us, none of the provided formats are correct. (We use either yy. mm. dd., or yyyy. MMMM dd.)
